examples/Node/file.js

```diff
@@ -21,8 +21,8 @@ detector.on('silence', function () {
   console.log('silence');
 });
 
-detector.on('sound', function () {
-  console.log('sound');
+detector.on('sound', function (buffer) {
+  console.log('sound', buffer);
```

Comment:
@gregorycuellar could you address the two comments I left here? I'll be ready to merge after that.